root/get_coverage_report.sh

```diff
@@ -0,0 +1,5 @@
+# Run with name as argument
```

Comment:
Use this script to pull the coverage report from the server for local viewing

run as: sh get_coverage_report.sh wimo